packages/buttons/demo/buttons.stories.js

```diff
@@ -6,65 +6,81 @@
  */
 
 import '../dist/index.css';
+import { Grid } from '@zendeskgarden/react-grid';
 import React from 'react';
+import { useTheme } from 'styled-components';
 
 export default {
   title: 'Components/Buttons'
 };
 
+const toClassName = (theme, base, size = false, style = false) => {
```

Comment:
Given that `style` is of type array, wouldn't `[]` as default be better suited?

```suggestion
const toClassName = (theme, base, size = false, style = []) => {
```

Execution after removing all styles.
![Screenshot 2024-10-08 at 9 31 59 AM](https://github.com/user-attachments/assets/74cc1434-fbb1-4cd7-a215-a40623f12a51)